### PR TITLE
Fix failure to proof hint so the link displays as a link

### PR DESCRIPTION
**Pull request by Jeremy1026**
### Relevant Ticket or Conversation:

[LG-11452](https://cm-jira.usa.gov/browse/LG-11452)

### Description of Changes:

Fix the Failure to Proof hint so that the link is clickable, instead of printing out the HTML.

### PR Checklist:

1. [ ] Have you linted and tested your code locally prior to submission?
2. [ ] Have you tagged the appropriate dev(s) for review?
3. [ ] Have you linked to any relevant tickets or conversations?

### PR Review Standards: 
- Consider using [Conventional Comments](https://conventionalcomments.org/) to ensure that your feedback is clear and actionable.
- Ideally, PRs should be reviewed by at least 2 team members.
- All PRs must be approved before being merged.

### CODE REVIEW

File: app/views/service_providers/_form.html.erb [hunk 1]
```diff
@@ -280,7 +280,7 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
                  input_html: { class: 'usa-input' },
                  label_html: { class: 'usa-input-required'},
                  label: 'Failure to Proof URL',
-                 hint: I18n.t('service_provider_form.failure_to_proof_url'),
+                 hint: I18n.t('service_provider_form.failure_to_proof_url').html_safe,
```
Comment (mitchellhenke) [inline app/views/service_providers/_form.html.erb:283]: I think we should prefer renaming the key of the YAML content to `failure_to_proof_url_html`, which should automatically take care of escaping. Even better would be to separate out the HTML/content, but that'd be more involved.

Comment (Jeremy1026) [reply app/views/service_providers/_form.html.erb:283]: Tried changing the name as suggested and dropping the html_safe, and that doesn't seem to do it automatically. I'll dig into it a bit to see whats going on.

Comment (mitchellhenke) [reply app/views/service_providers/_form.html.erb:283]: https://github.com/18F/identity-idp/blob/67df16d/config/locales/idv/en.yml#L264 is an example on the IDP side

**Review by nprimak — APPROVED** [review]

File: app/views/service_providers/_form.html.erb (hunk 1 above)
Comment (mitchellhenke) [reply app/views/service_providers/_form.html.erb:283]: I think it's as straightforward as using `t()`

Comment (mitchellhenke) [reply app/views/service_providers/_form.html.erb:283]: Oh I missed that this was merged

https://github.com/18F/identity-dashboard/compare/mitchellhenke/translation-helper shows the changes needed

**Pull request merged**